src/controllers/controllerengine.cpp

```diff
@@ -1440,6 +1440,22 @@ void ControllerEngine::softTakeover(QString group, QString name, bool set) {
 }
 
 /*  -------- ------------------------------------------------------
+     Purpose: Ignores the next value for the given ControlObject
+                This is used when an absolute physical control is changed to
+                to operate on a different ControlObject
+     Input:   ControlObject group and key values
+     Output:  -
+     -------- ------------------------------------------------------ */
+void ControllerEngine::softTakeoverIgnoreNext(QString group, QString name) {
```

Comment:
I would prefer Detach over Resync since the control is not in sync after the call. 
But I do not want to stress this any further, jut pick your favorite.